web/templates/web/faq.html

```diff
@@ -750,10 +777,15 @@ <h2 class="accordion-header" id="heading-29">
                  aria-labelledby="heading-29"
                  data-bs-parent="#faq-accordion-for-researchers">
                 <div class="accordion-body">
+                    <p>
+                        {% trans "Visit our" %}
+                        <a href="https://lookit.readthedocs.io/en/develop/researchers-start-here.html">{% trans "documentation" %}</a>
+                        {% trans "for a complete guide to posting your studies on Children Helping Science!" %}
+                    </p>
                     {% blocktranslate %}
                 <p>In order to list your study on this website, you first will need to have your institution sign a contract with MIT where they agree to the <a href="https://lookit.mit.edu/termsofuse">Terms of Use</a> and certify that their studies will be reviewed and approved by an institutional review board. You will also need to edit your IRB protocol to include online testing, or submit a new protocol for your proposed online study.</p>
-                <p>As of January 2022, we have agreements with over 50 universities, including institutions in the United States, Canada, the United Kingdom, and European Union. Please email <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a> if you are not sure whether your institution already has an agreement, or if you need any help at all getting your paperwork in order.</p>
-                <p>While you are completing these steps, you can get started on Lookit by creating a lab account, creating your first study, and getting it peer reviewed by our researcher community. A step-by-step guide to getting started is available <a href="https://lookit.readthedocs.io/en/develop/researchers-start-here.html">here</a>.</p>
+                <p>As of April 2023, we have agreements with over 90 universities, including institutions in the United States, Canada, the United Kingdom, and European Union. Please email <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a> if you are not sure whether your institution already has an agreement, or if you need any help at all getting your paperwork in order.</p>
```

Comment:
Same comment about the email address here: lookit@mit.edu vs lookit-tech@mit.edu. Should we use the same address?
(And I realize that this is how it was already, not something you changed!)

Reply:
Yup you found it!  I don't know why it was like that, but I'll make the change.